packages/cli/src/daemon-config.ts

```diff
@@ -0,0 +1,46 @@
+import { LoggerConfig, SyncOptions } from '@ceramicnetwork/common'
+
+export interface DaemonIpfsConfig {
```

Comment:
Can we add detailed TS docs in this file? Will be super nice to be able to just point to the generated docs for how to configure your node!

Reply:
Thanks for adding this!